index.js

```diff
@@ -2,7 +2,7 @@ if (process.env.NODE_ENV !== 'production') {
   require('dotenv').config();
 }
 const express = require('express');
-const app = express();
+const app = new express();
```

Comment:
New is not required as per the documentation, why are we adding it here? https://expressjs.com/en/starter/hello-world.html

Reply:
You're right! I found it checking other examples, and I forgot to remove it. Good catch!
```suggestion
const app = express();
```